Lib/distutils/tests/test_check.py

```diff
@@ -99,6 +104,11 @@ def test_check_restructuredtext(self):
         cmd = self._run(metadata, strict=1, restructuredtext=1)
         self.assertEqual(cmd._warnings, 0)
 
+        # check that includes work to test #31292
+        metadata['long_description'] = 'title\n=====\n\n.. include:: {}'.format(INCLUDE_TEST_PATH)
+        cmd = self._run(metadata, strict=1, restructuredtext=1)
+        self.assertEqual(cmd._warnings, 0)
```

Comment:
Is it possible to test the value of the long description (to see the desired text included)?